.github/workflows/health-sdk.publish.example.apps.firebase.yml

```diff
@@ -1,7 +1,13 @@
 name: Publish GiniHealthSDKExample to Firebase Distribution
 on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - 'HealthSDK/**'
+      - 'HealthAPILibrary/**'
```

Comment:
[Suggestion]: @ValentinaIancu-Gini  we need to add here `GiniInternalPaymentSDK` and `GiniUtilites`